scripts/software-versions-inventory/README.md

```diff
@@ -0,0 +1,55 @@
+# Script: Software Versions Inventory
+
+## Prerequisites
+
+### `cargo`
+
+Install `cargo` by installing the Rust toolchain via [rustup](https://rustup.rs).
+
+### `make`
+
+If you do not have it installed, install `make` on your system.
+On fedora, run `sudo dnf install make`.
+
+### `rpmspec`
+
+On fedora, run `sudo dnf install rpm-build` to install the `rpmspec` command, which is used in the software versions inventory script.
```

Comment:
IIRC, I was able to run this on a mac. Maybe instructions for brew? (https://formulae.brew.sh/formula/rpm ? it's been a while)

Reply:
Added `brew` instructions